app/src/main/res/layout/help_fragment.xml

```diff
@@ -2,6 +2,15 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto">
 
+  <data>
+
+    <import type="org.oppia.app.R" />
```

Comment:
Doesn't appear to be needed--please remove.

Reply:
Done.